### PR TITLE
feat: stripe integration

### PR DESCRIPTION
**Pull request by DimejiAre**
## Description
Describe change or new feature here.
## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
## Checklist
Remove any items which are not applicable.
- [x] I have performed a self-review of my own code
- [ ] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works .
- [ ] New and existing unit tests pass locally with my changes
## Link to Trello card
https://trello.com/c/Ndn12nsj

### CODE REVIEW

**Comment by DimejiAre** [comment]
[API endpoint for handling purchases / payment transaction](https://trello.com/c/Ndn12nsj/97-api-endpoint-for-handling-purchases-payment-transaction)

**Comment by DimejiAre** [comment]
[Buyer: Front end card payment flow(Stripe)](https://trello.com/c/SPMSeToV/107-buyer-front-end-card-payment-flowstripe)

File: routes/paymentRouter.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+// See your keys here: https://dashboard.stripe.com/account/apikeys
+const router = require('express').Router()
+
+const stripe = require('stripe')(process.env.STRIPE_SECRET);
+
+router.post('/charge', async (req,res) => {
+    let transaction = req.body
+    const paymentIntent = await stripe.paymentIntents.create({
+        amount: transaction.amount,
+        currency: transaction.currency || 'usd',
```
Comment (shaunorpen) [inline routes/paymentRouter.js]: Is it a good idea to default to USD if the currency isn't specified? I'm concerned that if I want to buy a pair of boxers for N500 I might get charged USD500 if the store isn't set up right!

**Review by shaunorpen — COMMENTED** [review]
Just a thought on default currencies before approving...one to discuss?

Also, please fill in type of change and other checklist items.

File: routes/paymentRouter.js (hunk 1 above)
Comment (DimejiAre) [reply routes/paymentRouter.js]: that's true. Will fix

**Review by shaunorpen — APPROVED** [review]

**Pull request merged**